src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.RSA.cs

```diff
@@ -69,8 +70,8 @@ internal static partial class AppleCrypto
             out SafeCFDataHandle pEncryptedOut,
             out SafeCFErrorHandle pErrorOut);
 
-        [LibraryImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_RsaDecryptPkcs")]
-        private static partial int RsaDecryptPkcs(
+        [LibraryImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_RsaDecryptRaw")]
```

Comment:
`AppleCryptoNative_RsaDecryptPkcs` is effectively dead code at this point. We can remove the native code and associated `DllImportEntry`, but if you want to hang on to it for whatever reason, that's fine.

Reply:
My hope is that we can jump back to it at some point, so I was leaving it easy.